dev-client/src/auth/index.ts

```diff
@@ -80,13 +80,16 @@ async function exchangeToken(
   identityJwt: string,
   provider: Omit<AuthProvider, 'google'> | `google-${'ios' | 'android'}`,
 ) {
+  let body: any = {
+    provider,
+    jwt: identityJwt,
+  };
+  if (provider === 'apple') {
```

Comment:
Question: Do you also need to check if the platform is ios?

```suggestion
  if (provider === 'apple' && Platform.OS === 'ios') {
```